### PR TITLE
build docker image

### PR DESCRIPTION
**Pull request by tomaszduda23**
Make multiarch docker image.

### CODE REVIEW

**Comment by tomaszduda23** [comment]
@mdevaev any preferences where you would like to put this image? It is not fully tested yet but it seems to be pretty much all.

File: .github/workflows/docker-alpine-image.yaml [hunk 1]
```diff
@@ -21,5 +31,41 @@ jobs:
...
+      -
+        name: Build multi arch
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          platforms: linux/arm64,linux/amd64,linux/arm/v7
+          file: pkg/docker/Dockerfile.alpine
+      -
+        name: Login to DockerHub
+        if: startsWith(github.ref, 'refs/tags/v')
```
Comment (tomaszduda23) [inline .github/workflows/docker-alpine-image.yaml:56]: Only tags should be push. Need to check if this is enough for checking.

**Comment by mdevaev** [comment]
I think we could push it to pikvm repo as you suggested

**Comment by tomaszduda23** [comment]
Image will be pushed to dockerhub when you create new tag which starts with 'v'. DOCKERHUB_USERNAME, DOCKERHUB_TOKEN has to be set in repo secrets.

**Pull request merged**